scripts/typesFiles.js

```diff
@@ -29,9 +31,17 @@ const createPropsList = ({ info }) => {
 
   /* prettier-ignore */
   const list = [`${declareComp}
-  interface ${displayName}Props {`,
+  interface ${displayName}Props{
+    [x:string]: any;
+    `,
   ];
 
+  /**
```

Comment:
**NOTE:** we decided to use the `[x:string]` approach over React.HTMLAttribute to reduce the number of suggestion whenever the consumer wants to see the component's prop.